### ISSUE TITLE
A folder is shown as empty if its name contains a "+"

### ISSUE THREAD

**Issue by pixelk0** [roundcube/roundcubemail#6997]
Roundcube 1.4rc2
filesystem_attachments | 1.0 |
filters | dev-master |
hide_blockquote | 1.0 |
jqueryui | 1.12.0 | 
markasjunk | 2.0 |
newmail_notifier | 0.8 | 
show_additional_headers | 2.0 |
zipdownload | 3.4 |

Navigating inside a folder that has a "+" (plus sign, encoded as %2B in the URL) shows the folder as being empty, even if it contains messages. If you rename the folder and remove the "+" from the name and navigate again into it, all the messages appear.
This wasn't the case in previous version of RC (sorry can't be more precise on the versions that worked).

**Comment by alecpl** [comment]
Works for me. Try with disabled filters plugin. Check log or browser console for errors.

**Comment by pixelk0** [comment]
Tried without the filters plugin, same result. Tried on two different servers, same result.
Nothing in lighttpd logs, nothing in the browser console. But I noticed that on every automatic refresh of the folder (every minute or so) an error message pops up, which makes me think somewhere RC isn't looking for the right folder :  
Error message : 
![ScreenCap-20191105-102640](https://user-images.githubusercontent.com/5732483/68195548-59129980-ffb7-11e9-92cd-b691e7a74435.png)
THe folder : 
![ScreenCap-20191105-102606](https://user-images.githubusercontent.com/5732483/68195560-5d3eb700-ffb7-11e9-8590-07b023f1ea9c.png)
The URL : 
![ScreenCap-20191105-102708](https://user-images.githubusercontent.com/5732483/68195564-5fa11100-ffb7-11e9-9563-615f0ce8168c.png)

If I rename this folder to FNAC without the "+", all the messages re-apear, it occurs on every folder containing a + in its name.
